connectors/es/sink.py

```diff
@@ -546,6 +546,10 @@ async def get_access_control_docs(self, generator):
         try:
             async for doc in generator:
                 doc, _, _ = doc
+
+                if doc is None:
+                    continue
```

Comment:
Should it be here? We should not yield None IMO

Reply:
I think it should be there as otherwise every connector implementation needs to never yield `None`, which is something out of the framework's control. Maybe log a warning/debug message? WDYT?

Reply:
`yield None` is weird cause also you yield multiple things: `yield None, download_func, DELETE` - that's also weird. I'd honestly just disable it by default, connectors should do this check themselves.

Where did this situation happen for you in current work?

Reply:
Ran into it, when some SPO system user was yielded. Will add the guards to SPO and remove it from `sink`